net/adguardhome/files/adguardhome.config

```
@@ -0,0 +1,3 @@
+config adguardhome config
+	# Where to store persistent data by AdGuard Home
+	option workdir /tmp/adguardhome
```

```suggestion
	option workdir /var/adguardhome
```

`/var` is usually symlinked to `/tmp`, and if the user decides to move `/var` to persistent storage then this directory would also become persistent.